src/store/reducers/__tests__/__snapshots__/modal.spec.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Modal reducer returns the initial state 1`] = `
+exports['Modal reducer returns the initial state 1'] = `
```

Comment:
it's an auto generated file, don't think it makes sense to have lint changes here (even auto --fix) 🤔 . once the modal component changes - this file will be re-generated. 
It would be good to change this particular rule to something else